commcare_connect/templates/account/account_inactive.html

```diff
@@ -1,11 +1,13 @@
-{% extends "account/base.html" %}
+{% extends "account/base.tailwind.html" %}
 
 {% load i18n %}
 
 {% block head_title %}{% translate "Account Inactive" %}{% endblock %}
 
 {% block inner %}
-<h1 class="mb-4">{% translate "Account Inactive" %}</h1>
+  <div class="w-full min-h-[620px] h-fit">
```

Comment:
how responsive is this? Generally specifying measurements in pixels rather than than relative measurements can lead to weird results on unusual screen sizes. I am not sure if tailwind has a way around that, but am surprised to see absolute pixel measurements here